### PR TITLE
IN-553 Add Environment Indicators

### PR DESCRIPTION
**Pull request by fouedrao**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -101,6 +101,7 @@
         "drupal/oembed_providers": "^1.1",
         "drupal/entity_clone": "^1.0@beta",
         "drupal/back_to_top": "^2.0"
+        "drupal/environment_indicator": "^4.0"
```
Comment (johnrockweilercgi) [inline composer.json:104]: This line looks weird, not sure why the colon has a background.

File: composer.lock [hunk 2]
```diff
@@ -2887,69 +2887,6 @@
                 "source": "https://git.drupalcode.org/project/autosave_form"
             }
         },
-        {
-            "name": "drupal/back_to_top",
```
Comment (johnrockweilercgi) [inline composer.lock:2891]: Not sure why it's removing the back to top module from composer.lock, It's listed in composer.json, so think this needs to be added back in.

File: composer.json (hunk 1 above)
Comment (fouedrao) [reply composer.json:104]: Updating PR - there was a missing comma on the line above.

**Review by fouedrao — COMMENTED** [review]
PR needs updates

**Pull request merged**